### DISCUSSION TITLE
FFT stability

### DISCUSSION THREAD

**Discussion by qfan2** [Part-II-Computational-Physics/.github#9] [General]
Anyone has any insight into FFT stability? It always seems to diverge no matter what dt or n_points_space values I choose.
Thanks.

**Comment by Jamiewhbee** [comment]
I used the Euler method, so the region of stability is |1 - i v k dt| < 1, but v, k, and dt are real so the solution always diverges.

**Reply by NathanMagnan** [reply]
Careful, there are several Euler method. 
You are perfectly right about the explicit Euler method, it is always unstable (for this equation).
But the implicit Euler method is usually more stable than the implicit one.

**Reply by NathanMagnan** [reply]
implicit usually more stable than explicit*
